.github/workflows/selenium-noIngestion-tests.yml

```diff
@@ -16,12 +16,12 @@ name: Selenium No Ingestion Java CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, '0.[0-9]+.[0-9]+' ]
```

Comment:
One quick enhancement suggestion here, let's follow a standard yaml pattern with below snippet everywhere in CI files!
```
branches:
      - main
      - '0.[0-9]+.[0-9]+'
```